tests/antithesis/Makefile

```diff
@@ -28,9 +28,13 @@ antithesis-build-etcd-image-release-3.6: antithesis-build-etcd-image
 antithesis-build-etcd-image-main: REF=main
 antithesis-build-etcd-image-main: antithesis-build-etcd-image
 
+.PHONY: antithesis-build-config-image
+antithesis-build-config-image:
+	docker build -f ./Dockerfile.config . -t etcd-config:latest -t etcd-config:$(REF) --build-arg REF=$(REF)
+
 .PHONY: antithesis-docker-compose-up
 antithesis-docker-compose-up:
-	export USER_ID=$(USER_ID) && export GROUP_ID=$(GROUP_ID) && docker-compose up
+	export USER_ID=$(USER_ID) && export GROUP_ID=$(GROUP_ID) && export REF=$(REF) && docker-compose up
```

Comment:
Default REF is `main`, instead maybe we could not template it and depend latest being default?